### ISSUE TITLE
Error installing in project with pnpm package manager

### ISSUE THREAD

**Issue by evgeniyworkbel** [stylelint/stylelint#8721]
### What minimal example or steps are needed to reproduce the bug?

Run `npm init stylelint` as first step to install

### What minimal configuration is needed to reproduce the bug?

-

### How did you run Stylelint?

-

### Which Stylelint-related dependencies are you using?

-

### What did you expect to happen?

No problem with installing in projects with `pnpm` package manager

### What actually happened?

The following problems were reported: 

```
npm init stylelint
Need to install the following packages:
create-stylelint@0.5.0
Ok to proceed? (y) y


> likevolley@0.0.0 npx
> create-stylelint

✔ Created .stylelintrc.json.
✖ Failed to install packages:
ExecaError: Command failed with exit code 1: npm install -D stylelint stylelint-config-standard
```

### Do you have a proposal to fix the bug?

No

**Comment by jeddy3** [comment]
@evgeniyworkbel Thanks for the report.

The setup tool and Stylelint should be compatible with pnpm with `pnpm create stylelint`. I've opened a PR to revise some of the instructions in the getting started guide (https://github.com/stylelint/stylelint/pull/8726) to make them clearer.

While waiting for that PR to land, you can use the next version of the `create` tool with:

```console
pnpm dlx stylelint/create-stylelint#c0838f16b70bc6fbb148b5a456b8d5a00f42013d
```

If you accept the continue prompt, the output should look something like:

<img width="691" height="390" alt="Image" src="https://github.com/user-attachments/assets/717896c1-9e13-4c58-acc2-5cae3a257629" />

**Comment by evgeniyworkbel** [comment]
many thanks!
